db/migrate/20210802114054_add_notification_setting_options.rb

```diff
@@ -18,21 +17,4 @@ def add_notification_settings_options
       t.index :all
     end
   end
-
-  def update_notified_events
-    event_types = %w(work_package_added work_package_updated work_package_note_added status_updated work_package_priority_updated)
-
-    # rubocop:disable Rails/WhereExists
-    # The Setting.exists? method is overwritten
-    reversible do |dir|
-      dir.up do
-        Setting.notified_events = Setting.notified_events - event_types if Setting.where(name: 'notified_events').exists?
-      end
-
-      dir.down do
-        Setting.notified_events = Setting.notified_events + event_types if Setting.where(name: 'notified_events').exists?
-      end
-    end
-    # rubocop:enable Rails/WhereExists
-  end
```

Comment:
The setting could now be removed from the database altogether.

Reply:
Yes indeed :+1:

Reply:
👍